reporters/ReportVectorMigration.cpp

```diff
@@ -93,10 +98,16 @@ GetReportInstantiator( Kernel::instantiator_function_t* pif )
 #endif
 
     ReportVectorMigration::ReportVectorMigration()
-        : BaseTextReport( "ReportVectorMigration.csv" )
-        , m_StartDay( 365.0 )
-        , m_EndDay( 372 )
+        : BaseTextReport(DEFAULT_NAME)
+        , m_StartDay( 0)
+        , m_EndDay(FLT_MAX)
         , m_IsValidDay( false )
+        , m_MustBeInState()
+        , m_MustBeFromNode()
+        , m_MustBeToNode()
+        , m_IncludeGenomeData( false )
+        , m_SpeciesList()
+        , m_FilenameSuffix()
```

Comment:
Is it better that we copy all this filtering stuff here or someone use ReportFilter?  StartDay, EndDay, FilenameSuffix are all in ReportFilter.  I also wonder if we had a ReportFilterVector, could we standardize our report filtering.

Reply:
made it a TODO: https://github.com/InstituteforDiseaseModeling/EMOD/issues/29